### PR TITLE
Ut afd is enabled

### PR DESCRIPTION
**Pull request by garimaprachi**

### CODE REVIEW

File: tests/phpunit/tests/test-affordabilitywidget.php [hunk 1]
```diff
@@ -84,4 +84,79 @@ public function testgetCustomisation()
...
+
+        $this->assertSame('true', isEnabled('rzp_afd_enable_dark_logo'));
+    }
+
+    public function testgetFooterDarkLogo()
+    {
+        $this->assertSame('true', getFooterDarkLogo());
+    }
+
+    public function testgetPayLater()
```
Comment (yashgit891) [inline tests/phpunit/tests/test-affordabilitywidget.php:102]: Write UT for paylater, cardless emi, emi when the limited_providers are not present.

Comment (garimaprachi) [reply tests/phpunit/tests/test-affordabilitywidget.php:102]: Added

File: tests/phpunit/tests/test-affordabilitywidget.php [hunk 2]
```diff
@@ -84,4 +84,85 @@ public function testgetCustomisation()
...
+    {
+        add_option('rzp_afd_enable_pay_later', 'yes');
+        
+        $this->assertTrue(true,getPayLater());
```
Comment (abdulwahidsharief) [inline tests/phpunit/tests/test-affordabilitywidget.php:106]: @garimaprachi leave space after , and can we use assertTrue

File: tests/phpunit/tests/test-affordabilitywidget.php [hunk 3]
```diff
@@ -84,4 +84,85 @@ public function testgetCustomisation()
...
+        $response = getPayLater();
+
+        $this->assertStringContainsString('"providers": ["getsimpl","icic",]', $response);
+    }
+
+    public function testgetCardlessEmi()
+    {
+        add_option('rzp_afd_enable_cardless_emi', 'yes');
+        
+        $this->assertTrue(true,getCardlessEmi());
```
Comment (abdulwahidsharief) [inline tests/phpunit/tests/test-affordabilitywidget.php:119]: @garimaprachi leave space after ,

File: tests/phpunit/tests/test-affordabilitywidget.php [hunk 4]
```diff
@@ -84,4 +84,85 @@ public function testgetCustomisation()
...
+        $response = getCardlessEmi();
+
+        $this->assertStringContainsString('"providers": ["hdfc","icic",]', $response);
+    }
+
+    public function testgetEmi()
+    {
+        add_option('rzp_afd_enable_emi', 'yes');
+        
+        $this->assertTrue(true,getEmi());
```
Comment (abdulwahidsharief) [inline tests/phpunit/tests/test-affordabilitywidget.php:132]: @garimaprachi leave space after ,

File: tests/phpunit/tests/test-affordabilitywidget.php [hunk 5]
```diff
@@ -140,4 +146,108 @@ public function testgetKeyID()
...
+    }
+
+    public function testgetFooterDarkLogo()
+    {
+        $this->assertSame('true', getFooterDarkLogo());
+    }
+
+    public function testgetPayLater()
+    {
+        add_option('rzp_afd_enable_pay_later', 'no');
```
Comment (abdulwahidsharief) [inline tests/phpunit/tests/test-affordabilitywidget.php:171]: @garimaprachi please create separate test cases for different scenarios

File: tests/phpunit/tests/test-affordabilitywidget.php [hunk 6]
```diff
@@ -140,4 +146,108 @@ public function testgetKeyID()
...
+    public function testisEnabled()
+    {
+        add_option('rzp_afd_enable_dark_logo', 'yes');
+        
+        $this->assertSame('true', isEnabled('rzp_afd_enable_dark_logo'));
+
+        delete_option('rzp_afd_enable_dark_logo');
+    }
+
+    public function testisEnabledOption()
```
Comment (abdulwahidsharief) [inline tests/phpunit/tests/test-affordabilitywidget.php:159]: @garimaprachi can we use more convenient function name

**Review by abdulwahidsharief — APPROVED** [review]
LGTM

**Pull request merged**